starter_notebook_reverse_training.ipynb

```diff
@@ -0,0 +1,1292 @@
+{
```

Comment:
<p>Please add a comment here that the reversal is on purpose ;)</p><p>The name variable can directly be changed to $trg_$src.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane-mt/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Alright Noted

Reply:
Didn't quite get this but I added a note at the beginning:)